packages/components/src/styles/_typography.scss

```diff
@@ -0,0 +1,5 @@
+
+$a8c-fontFamily-sans: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell',
+'Helvetica Neue', sans-serif;
```

Comment:
This line break with no indentation doesn't look right. Was that manual or automated?